workers/generic-worker/multiuser.go

```diff
@@ -192,7 +207,7 @@ func deleteExistingOSUsers() (err error) {
 	}
 	allErrors := []string{}
 	for _, username := range userAccounts {
-		if strings.HasPrefix(username, "task_") && username != taskContext.User.Name && username != gwruntime.AutoLogonUser() {
+		if strings.HasPrefix(username, "task_") && username != taskContext.User.Name && username != nextTaskUser {
```

Comment:
probably a silly question - how does it know which task is next? if you have capacity>1, wouldn't there be multiple next users? 
Or this case is not handled yet (or handled in a different place)?

Reply:
That isn't a silly question, that is a good one. Under Generic Worker, you need one generic-worker process per unit capacity, so if you want to run 5 tasks in parallel, you would have 5 generic-worker processes running. The `nextTaskUser` is created ahead of time, so at this point in the code, the value has already been initialised.

Reply:
ah, got it!